code/modules/spells/aoe_turf/conjure/snakes.dm

```diff
@@ -19,7 +19,7 @@
 	delete_snakes()
 	if(!..())
 		user.visible_message("<span class='warning'>\The [user]'s body splits into a mass of snakes!</span>","<span class='notice'>Your body splits into a mass of snakes.</span>")
-		user.transmogrify(/mob/living/simple_animal/cat/snek/wizard, TRUE)
+		user.transmogrify(/mob/living/simple_animal/cat/snek/wizard, TRUE, FALSE)
```

Comment:
I've deliberately left the tweaks at the top of the changelog to be the first thing anyone will read because of how many changes have accumulated, but if even a single person is against Snake Form not killing the user (including you) I will remove it from the PR and make a separate PR. I think it'll be the safe to leave it in the PR otherwise.